korebuild.json

```diff
@@ -1,11 +1,11 @@
 {
   "$schema": "https://raw.githubusercontent.com/aspnet/BuildTools/dev/tools/korebuild.schema.json",
-  "channel": "dev",
+  "channel": "release/2.1",
```

Comment:
Using the release version of the tools, which is going to track 2.1 and its patches. If we retarget at later builds, we can update then. I figured we don't want to use 2.2 yet.